packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -18,31 +21,45 @@ class Panel extends React.Component {
   }
 
   render() {
+    const {
+      isFramed,
+      isToggleable
+    } = this.props
+
     return (
-      <div>
+      <StyledPanel
+        isFramed={isFramed}
+      >
         {
           React.Children.map(this.props.children, child =>
             React.cloneElement(child, {
+              isFramed: isFramed,
               isOpen: this.state.isOpen,
-              isToggleable: this.props.isToggleable,
+              isToggleable: isToggleable,
               toggleOpenState: this.toggleOpenState
             })
           )
         }
-      </div>
+      </StyledPanel>
     )
   }
 }
 
 Panel.defaultProps = {
+  isFramed: true,
   isOpen: false,
   isToggleable: true
 }
 
 Panel.propTypes = {
   children: PropTypes.node,
   /**
-   * Boolean to control if <PanelBody/> is initially opened. Default value is 'false'.
+   * Boolean to control if <Panel.Header/>, <Panel.Body/> and <Panel.Footer/> is initially opened.
+   * Default value is 'false'.
```

Comment:
- Really to control if [...] is "initially openend"? Why is the prop called "isFramed"?
- The default value on line 49 is `true`, not `false`

Reply:
Descriptions were wrong.